.travis.yml

```diff
@@ -43,10 +48,9 @@ before_install:
   # Set up the Groovy environment
   - export GROOVY_HOME=/usr/share/groovy/
 
-cache:
-  directories:
-    - $HOME/android-ndk-linux
-    - $HOME/android-sdk-linux
```

Comment:
I remember the reason we cached these is that it took so long to install them it would cause jobs to time out.